backend/annotation/views.py

```diff
@@ -25,16 +25,16 @@ def annotation_handler(request):
         if token and authentication.is_authenticated(token):
             try:
                 user_id = authentication.get_user_id(token)
-                request_data = request.data
+                request_data = json.loads(request.body.decode('utf-8'))
                 targets = []
                 if 'targets' in request_data:
                     targets = request_data['targets']
                 body = request_data['body'] if 'body' in request_data else None
                 annotation = Annotation()
-                annotation.context = request_data['context']
+                annotation.context = request_data['context'] if 'context' in request_data else None
```

Comment:
Let's not say ``None`` to it so it takes default value if not specified.

Reply:
context currently does not have a default value

Reply:
Please add this: ``http://www.w3.org/ns/anno.jsonld``